packages/eas-cli/src/metadata/utils/ajv.ts

```diff
@@ -0,0 +1,27 @@
+import { AggregateAjvError, HumanError } from '@segment/ajv-human-errors';
+import Ajv, { ErrorObject } from 'ajv';
+import addFormats from 'ajv-formats';
+
+/**
+ * Create a new AJV validator using the JSON Schema 06 draft.
+ * It also adds the additional formats from the `ajv-formats` package.
+ *
+ * @see https://github.com/ajv-validator/ajv-formats
+ */
+export function createValidator(): Ajv {
+  const validator = new Ajv({
+    strict: false, // The metadata schema is shared with vscode, including vscode-only properties
+    verbose: true, // Required for `ajv-human-errors`
+    allErrors: true, // Required for `ajv-human-errors`
+  });
+
+  return addFormats(validator).addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'));
```

Comment:
Move the require to the top-level scope. This way if something changes in `ajv`, we can know about it earlier than when calling the function.

Reply:
@wkozyra95 made a good point about the EAS CLI not being lazy-loaded ([see comment](https://github.com/expo/eas-cli/pull/1416#discussion_r993154086)). I think it doesn't make sense to load these JSON files every time, but I don't have a strong opinion on that though.

Reply:
I was thinking there about calling compile in the global scope(I'm not sure how heavy it is). Importing in global scope should be fine.

Reply:
Ah gotcha. Well, in that case, we can import the JSON files top level. I will make that change after I split up the metadata context.